example/express.js

```diff
@@ -2,14 +2,33 @@ var express = require("express");
 var alexa = require("../index.js");
 var bodyParser = require("body-parser");
 
-var app = express();
+
 var PORT = process.env.port || 8080;
+var app = express();
+
+// ALWAYS setup the alexa app and attach it to express before anything else.
+var alexaApp = new alexa.app("test");
+
+// checkCert verifies requests come from amazon alexa. Must be enabled for production.
+// you can disable this if you're running a dev environment and want to POST things
+// to test behavior. enabled by default.
+var checkCert = false
+
+// enableDebug sets up a GET route when set to true. This is handy for testing in 
+// development, but not recommended for production. disabled by default
+var enableDebug = true
 
+alexaApp.express(app, express.Router(), checkCert, enableDebug);
```

Comment:
We call this `app` here, but maybe to be consistent it should be called `express`?